package/windows/Dockerfile.20H2

```diff
@@ -2,7 +2,7 @@ ARG SERVERCORE_VERSION
 
 FROM mcr.microsoft.com/windows/servercore:${SERVERCORE_VERSION} as download
 ENV ARCH=amd64
-ENV WMI_EXPORTER_ARCHIVER_VERSION 0.12.0
+ENV WINDOWS_EXPORTER_ARCHIVER_VERSION 0.15.0
```

Comment:
Note: wins version will be updated in a separate PR. For now, this will still be using the older version of wins (v0.0.4), but once https://github.com/rancher/wins/pull/14 and https://github.com/rancher/wins/pull/19 are released and tagged, I'll create another PR for this repository